README.md

```diff
@@ -2,15 +2,21 @@
 
 A complete study plan to become a Machine Learning Engineer with links to all FREE resources. If you finish the list you will be equipped with enough theoretical and practical experience to get started in the industry! I tried to limit the resources to a minimum, but some courses are extensive.
 
-Watch the video on YouTube for instructions:  
-[![Alt text](https://img.youtube.com/vi/dYvt3vSJaQA/hqdefault.jpg)](https://www.youtube.com/watch?v=dYvt3vSJaQA)  
-[https://www.youtube.com/watch?v=dYvt3vSJaQA](https://www.youtube.com/watch?v=dYvt3vSJaQA)
+### Watch the video on YouTube for instructions:  [Link to Youtube Video](https://www.youtube.com/watch?v=dYvt3vSJaQA)
 
-#### IMPORTANT: 
+<br>
+
+<p align="center">

```

Comment:
nice work! but another eralier PR has been merged that conflicts with the centered div here. I prefer the current smaller image, so I suggest just remove this here